### ISSUE TITLE
Line Endings 

### ISSUE THREAD

**Issue by tritowntim** [halfzebra/create-elm-app#2]
Very cool project, I'm excited to use it!

But having trouble using it on a Mac...

```shell
create-elm-app app
env: node\r: No such file or directory
```

Looked like a [line ending issue] (http://stackoverflow.com/questions/30344858/node-script-executable-not-working-on-mac-env-node-r-no-such-file-or-directo)...
Fixed the line endings in the CLI file:

```shell
dos2unix create-elm-app-cli.js 
dos2unix: converting file create-elm-app-cli.js to Unix format...
```

And that seemed to do the trick:

```shell
➜  create-elm-app app2

Creating app2 project...

Starting downloads...

  ● elm-lang/virtual-dom 1.1.1
  ● elm-lang/html 1.1.0
  ● elm-lang/core 4.0.5

Packages configured successfully!

Project is successfully created in `/Users/tritowntim/code/tmp/app2`.
``` 

I would submit a PR, but can't get git to recognize the new file endings.

**Comment by halfzebra** [comment]
Thank you for your submission! :+1: 

I will fix that during this weekend, along with many other improvements.

**Comment by halfzebra** [comment]
This is fixed with [0.1.3](https://github.com/halfzebra/create-elm-app/tree/0.1.3)
